os/kernel/binary_manager/binary_manager_load.c

```diff
@@ -282,58 +292,114 @@ static int binary_manager_load_all(void)
 	return BINMGR_OPERATION_FAIL;
 }
 
-static void reload_kill_each(FAR struct tcb_s *tcb, FAR void *arg)
+static int binary_manager_kill_task(FAR struct tcb_s *tcb)
```

Comment:
Could you let me know the reason why you use new function instead of task_terminate?

Reply:
I made a new function with necessary functions of task_terminate.
That's because these threads are already inactivated by fault recovery at this time.

Reply:
This is not binary manager operation. Let's move this into sched.